phoenix-core/src/it/java/org/apache/phoenix/end2end/json/JsonFunctionsIT.java

```diff
@@ -344,6 +344,68 @@ public void testJsonExpressionIndexInvalid() {
                 " (JSON_MODIFY(jsoncol, '$.info.tags[2]', '\"newValue\"')) include (col)");
     }
 
+    @Test
```

Comment:
Can you add a testcase where there is a filter/conditon on a JSON_EXISTS predicate.

Reply:
sorry I didn't understand this part. Can you give an example? @tkhurana

Reply:
I couldn't find a matching explanation in the JSON Standard.